reliability-engineering/pipelines/concourse-deployer.yml

```diff
@@ -21,7 +21,7 @@ resources:
   icon: github
   type: git
   source:
-    branch: ((deployment_branch))
```

Comment:
It's probably better to leave this parametrized, even though it's a bit weird needing to create two parallel branches. If we're going to fix this we should probably split it into two variables, one for each repo.